### DISCUSSION TITLE
Ed's Errno 2

### DISCUSSION THREAD

**Discussion by eddhyne** [community/community#66781] [Programming Help]
### Body

I am following "DJANGO for PROFESSIONALS" By William S. Vinvent and in Chapter 2: Docker Hello, World, I enter docker-compose up and I get: ch2-hello-web-1  | python: can't open file '/code/manage.py': [Errno 2] No such file or directory
ch2-hello-web-1 exited with code 2. See my code attached below. Please help.
[Errno2.docx](https://github.com/community/community/files/12578129/Errno2.docx)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by seyLu** [comment]
Hi @eddhyne,

![image](https://github.com/community/community/assets/98249191/13d90b87-ca0e-4b50-a1a0-2484941c31ac)

This is taken from the official Docker docs for Django and Docker Compose:
```dockerfile
# Dockerfile

# Choose the appropriate Python version
FROM python:3.x  

ENV PYTHONDONTWRITEBYTECODE=1
ENV PYTHONUNBUFFERED=1

# Set the working directory in the container
WORKDIR /code

# Copy the requirements file first to leverage Docker caching
COPY requirements.txt /code

# Install dependencies
RUN pip install -r requirements.txt

# Copy the entire project into the container
COPY . /code
```

You can read more on:
- [Docker sample Django](https://docs.docker.com/samples/django/)
- [Docker documentation sample for Django](https://github.com/docker/awesome-compose/blob/master/django/app/Dockerfile)
